src/dataframe/io.jl

```diff
@@ -814,11 +814,11 @@ function readtable(io::IO,
                    skipstart::Integer = 0,
                    skiprows::AbstractVector{Int} = Int[],
                    skipblanks::Bool = true,
-                   encoding::Symbol = :utf8,
+                   encoding::String = "UTF-8",
                    allowescapes::Bool = false,
                    normalizenames::Bool = true)
-    if encoding != :utf8
-        throw(ArgumentError("Argument 'encoding' only supports ':utf8' currently."))
+    if !any(encodings().==encoding)
```

Comment:
Just `!(encoding in encodings())`. But maybe we don't need to check this at all, and leave `StringDecoder` throw the error (if it's explicit enough)?